setup.py

```diff
@@ -59,7 +59,7 @@ def _dbt_trino_version():
         ]
     },
     install_requires=[
-        "dbt-core=={}".format(dbt_version),
```

Comment:
What about `dbt-core~={}` ?

Reply:
Yes! We recommend using `~=` (compatibility operator)